dimod/core/sampler.py

```diff
@@ -238,3 +239,23 @@ def sample_qubo(self, Q, **parameters):
         """
         bqm = BinaryQuadraticModel.from_qubo(Q)
         return self.sample(bqm, **parameters)
+
+    def validate_kwargs(self, **kwargs):
```

Comment:
I am worried that this is too general a name. I might expect it to validate the correctness of my kwargs. Maybe `validate_kwarg_set`?

Reply:
What about something more descriptive like `remove_unknown_kwargs()`?